src/models/generated-report.ts

```diff
@@ -23,44 +17,59 @@ export class GeneratedReport {
   public org: string;
   public status: string;
   public indexFile: IndexFile;
-  public userInfo: ReportUserInfo;
+  public userInfo: User = new User();
+
+  public template: string;
+  public args: Object;
+  public groupSlug: string;
+  public token: string;
 
   constructor(data: any = {}) {
     this.id = data['id'] || '';
     this.label = data['label'] || '';
     this.sourceRef = data['source_ref'] || '';
     this.url = data['url'] || '';
```

Comment:
Please remove the `url` field as it is not needed, and I will like to remove from API as well